src/lib/core/styles/tokens/avatar/_tokens.scss

```diff
@@ -0,0 +1,24 @@
+@use 'sass:map';
+@use '../../animation';
+@use '../../border';
+@use '../../elevation';
+@use '../../shape';
+@use '../../spacing';
+@use '../../theme';
+@use '../../typography';
+@use '../../utils';
+
+$tokens: (
+  size: utils.module-val(avatar, size, 40px),
+  font-size: utils.module-val(avatar, font-size, 1rem),
+  font-weight: utils.module-val(avatar, font-weight, 400),
```

Comment:
I think I agree with letting the standard typography tokens handle it, because those always specify size/weight/letter-spacing/line-height together, and if we just allow setting size/weight via variables here, the other values might be off.  If you have to override the font properties, it makes sense to consider them holistically, which I feel like is better encouraged by not providing these at the component level.